ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -264,72 +270,74 @@ public void shouldFilterNullPriorCommand() {
   }
 
   @Test
-  public void shouldDistributeCommand() throws ExecutionException, InterruptedException {
-    final KsqlConfig ksqlConfig = new KsqlConfig(
-        Collections.singletonMap(KsqlConfig.KSQL_PERSISTENT_QUERY_NAME_PREFIX_CONFIG, "foo"));
-    final Map<String, Object> overrideProperties = Collections.singletonMap(
-        ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
-    final String statementText = "test-statement";
-
-    final Statement statement = mock(Statement.class);
-    final Capture<ProducerRecord<CommandId, Command>> recordCapture = Capture.newInstance();
-    final Future<RecordMetadata> future = mock(Future.class);
-
-    expect(commandProducer.send(capture(recordCapture))).andReturn(future);
-    future.get();
-    expectLastCall().andReturn(null);
-    replay(commandProducer, future);
+  public void shouldDistributeCommand() {
+    ArgumentCaptor<ProducerRecord<CommandId, Command>> recordCaptor =
```

Comment:
Make it a field using `@Captor`:

```java
@Captor
  private ArgumentCaptor<ProducerRecord<CommandId, Command>> recordCaptor;
```

Which also fixes the unchecked cast warning.